backend/fms_core/models/container.py

```diff
@@ -87,14 +87,9 @@ def add_error(field: str, error: str):
                 if not errors.get("coordinates") and not errors.get("location") \
                         and not parent_spec.coordinate_overlap_allowed:
                     # Check for coordinate overlap with existing child containers of the parent
-                    try:
-                        check_coordinate_overlap(self.location.children, self, self.location)
```

Comment:
This is a risky change just to make your debugger happy. I would recommend removing the change from this PR and moving it to its own branch for future inclusion and creating a ticket so it gets properly tested when we have some time.